moveit_ros/moveit_servo/src/servo_calcs.cpp

```diff
@@ -595,7 +593,6 @@ void ServoCalcs::composeJointTrajMessage(const sensor_msgs::JointState& joint_st
                                          trajectory_msgs::JointTrajectory& joint_trajectory) const
 {
   joint_trajectory.header.frame_id = parameters_.planning_frame;
```

Comment:
I think the same comment as well as explicitly setting the stamp to 0 would be good here:

// When a joint_trajectory_controller receives a new command, a stamp of 0 indicates "begin immediately"
// See http://wiki.ros.org/joint_trajectory_controller#Trajectory_replacement
initial_joint_trajectory->header.stamp = 0;